.gitlab-ci.yml

```
@@ -77,43 +72,38 @@ test:format:
 
 
 
-test:checks:
+test:unittests:
   <<: *sync_remote_branches
   stage: test
   script:
-    - apt-get -qq update && apt-get -qqy install --allow-unauthenticated e2tools
-    # Install code coverage tooling
-    - GO111MODULE=off go get -u github.com/axw/gocov/gocov
-    - GO111MODULE=off go get -u golang.org/x/tools/cmd/cover
-    - make get-tools
+    - make get-go-tools
     - make check
```

`check` does not exist any more. See [CI output](https://gitlab.com/Northern.tech/Mender/mender-cli/-/jobs/806765745).

I guess we want:
```suggestion
    - make test-unit
```